brocade-vdx/workflows/brocade-vdx-config-deploy-graph.json

```diff
@@ -0,0 +1,30 @@
+{
+    "friendlyName": "Deploy Brocade VDX switch",
+    "injectableName": "Graph.Switch.BrocadeVDX.Deploy",
+    "options": {
+        "deploy-startup-config": {
+            "startupConfig": null
+        }
+    },
+    "tasks": [
+        {
+            "label": "deploy-startup-config",
+            "taskDefinition": {
+                "friendlyName": "Deploy switch config",
+                "injectableName": "Task.Inline.Switch.Deploy.Brocade",
+                "implementsTask": "Task.Base.Linux.Commands",
+                "options": {
+                    "startupConfig": null,
+                    "startupConfigUri": "{{ api.base }}/templates/{{ options.startupConfig }}",
+                    "commands": [
+                        {
+                            "downloadUrl": "/api/1.1/templates/brocade-vdx-deploy-config.py"
+                        }
```

Comment:
@dpasupathi The 1.1 API will be deprecated in the near future.  We should be using `/api/current/templates` instead.